rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -143,6 +143,9 @@ public class BridgeSupport {
     // (6 blocks/hour, 24 hours/day, 30 days/month)
     public static final Integer BTC_TRANSACTION_CONFIRMATION_MAX_DEPTH = 4320;
 
+    public static final int BRIDGE_BTC_TX_VERSION_2 = 2;
+    public static final int BRIDGE_BTC_TX_LEGACY_VERSION = 1;
```

Comment:
Not sure it's worth creating constants for this

Reply:
Ok, but keep in mind these values are also used in ReleaseTransactionBuilder class. So maybe the constants should go there?

Reply:
Done. Constants moved.